### PR TITLE
Added a notebook that tests the mlflow tracking feature that is set to release in 1.6.0

### PR DESCRIPTION
**Pull request by amit-elbaz**

### CODE REVIEW

File: test-mlflow-tracking/test-mlflow-tracking.ipynb [hunk 1]
```diff
@@ -0,0 +1,1906 @@
+{
```
Comment (MichaelSkralivetsky) [inline test-mlflow-tracking/test-mlflow-tracking.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    ! pip uninstall mlrun -y
</pre><p>no need, remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/test-notebooks/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (MichaelSkralivetsky) [inline test-mlflow-tracking/test-mlflow-tracking.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    ! pip install mlrun==1.6.0-rc11
</pre><p>no need, remove, this anyway will be running only on 1.6.0-rc12 and newer during automatic tests</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/test-notebooks/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Review by MichaelSkralivetsky — CHANGES_REQUESTED** [review]
no need to uninstall/install mlrun, automation is taking care of this

**Comment by amit-elbaz** [comment]
@MichaelSkralivetsky Fixed

**Pull request merged**